packages/dashboard/src/lib/journeys-utils.ts

```
@@ -0,0 +1,441 @@
+  sortBy: JourneyFilterState['sortBy'],
+  sortOrder: JourneyFilterState['sortOrder']
+): JourneyData[] {
+  return [...journeys].sort((a, b) => {
+    let comparison = 0
+
+    switch (sortBy) {
+      case 'startTime':
+        comparison = a.startTime - b.startTime
+        break
+      case 'endTime':
+        const aEndTime = a.endTime || Date.now()
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)